fdbclient/VersionedMap.h

```diff
@@ -483,7 +483,7 @@ class VersionedMap : NonCopyable {
 		return r->second;
 	}
 
-	static const int overheadPerItem = 128*4;
+	static const int overheadPerItem = 96*4;
```

Comment:
There's not really any documentation for how this overhead is getting calculated, but I assume you have some idea since you're changing it. If so, would you mind adding a comment to explain what this is based on? As an aside, if it's based on the fast allocated size of `PTree` objects, it seems like that wouldn't be fixed given that some of the types involved are templated. It looks like we only use this variable in one place where the types would be fixed, so maybe that's how we'd get away with it?